libirods_rule_engine_plugin-indexing.cpp

```diff
@@ -697,6 +740,151 @@ namespace {
         return retvalue.size() ? retvalue : "application/octet-stream";
     }
 
+    class PermsCalc {
```

Comment:
use lowercase letters and underscores

consider using more letters (e.g. `permissions_calculator`?)